web/ugahacks-6/.gitignore

```diff
@@ -32,3 +32,9 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+#yarn
```

Comment:
Why did you add this to the gitignore? You shouldn't be installing directly to a directory. You should be using the workspace syntax.

Reply:
Yes, I had a meeting with Andrei and started to use yarn workspace add <package>. Previously, I was incorrectly using yarn install and npm install directly.